### PR TITLE
243 email progress saved

### PR DESCRIPTION
**Pull request by asmega**
- depends on https://github.com/ministryofjustice/fb-submitter/pull/25

### CODE REVIEW

File: docs/v1/swagger.json [hunk 1]
```diff
@@ -136,17 +136,42 @@
                 "encrypted_details": {
                   "type": "string",
                   "example": "encrypted:payload"
+                },
+                "email": {
+                  "type": "object",
```
Comment (solidgoldpig) [inline docs/v1/swagger.json]: Add `template_name`

File: spec/integration/save_return_spec.rb [hunk 2]
```diff
@@ -20,6 +20,14 @@
         properties: {
           encrypted_email: { type: :string, example: 'encrypted:user@example.com' },
           encrypted_details: { type: :string, example: 'encrypted:payload' },
+          email: {
+            type: :object,
+            properties: {
```
Comment (solidgoldpig) [inline spec/integration/save_return_spec.rb]: Add `template_name`

Comment (asmega) [reply spec/integration/save_return_spec.rb]: done

File: docs/v1/swagger.json (hunk 1 above)
Comment (asmega) [reply docs/v1/swagger.json]: done

**Pull request merged**